portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** User comment and the email address the comment was written by. */
+public class Comment {
+
+  private final String text;
+  private final String email;
+
+  public Comment(String email, String text) {
```

Comment:
I could easily call this constructor with text and email in the wrong order. How could you prevent that as a class designer?

No need to make changes now, just something to start thinking about.

Reply:
ToTT `Cleanly Create Test Data` might help on this.